### DISCUSSION TITLE
NREL 5MW in HAWC2 vs OpenFAST only aerodynamic models

### DISCUSSION THREAD

**Discussion by LaurenceWETI** [OpenFAST/openfast#1614] [Q&A] [answered]
Dear everyone,

as we tried to run the [5MW_Land_BD_DLL_WTurb](https://github.com/OpenFAST/r-test/tree/ef6ab575919f1db597702f1c3f86a16bdc2b4338/glue-codes/openfast/5MW_Land_BD_DLL_WTurb) test in HAWC2 and OpenFAST/BeamDyn, we noticed deviations in our results, even when only the aerodynamic model is activated. The simulation is performed with deactivated ServoDyn, Gravity=0, steady windy speed at 11.6 m/s, constant rotational speed RotSpeed=12.1 rpm, and with deactivated Generator DOF. 

```
          2   CompElast   
          1   CompInflow  
          2   CompAero    
          0   CompServo   
          0   CompHydro   
          0   CompSub     
          0   CompMooring 
          0   CompIce     
          0   MHK         

```

```
True          FlapDOF1  
True          FlapDOF2  
True          EdgeDOF   
False         TeetDOF   
True          DrTrDOF   
False          GenDOF   
False          YawDOF   
True          TwFADOF1  
True          TwFADOF2  
True          TwSSDOF1  
True          TwSSDOF2  
False         PtfmSgDOF 
False         PtfmSwDOF 
False         PtfmHvDOF 
False         PtfmRDOF  
False         PtfmPDOF  
False         PtfmYDOF  

```

The figures bellow show the differences in the aerodynamic torque and the  rotational speed in both tools. 

![grafik](https://github.com/OpenFAST/openfast/assets/47774400/3b7b6b81-a701-490d-97aa-32ca50331702)

![grafik](https://github.com/OpenFAST/openfast/assets/47774400/d63fb8f2-117c-4e2d-a100-983597a84a7e)

Do anyone have any idea what could cause these differences?
Best regards

**Comment by jjonkman** [comment, marked as answer]
Dear @LaurenceWETI,

I would first suggest simplifying the model even further--I still see that you have structural flexibility of the rotor enabled (via BeamDyn), as well as drivetrain torsion and tower deflection.  Do the results between OpenFAST and HAWC2 compare better when only aerodynamics are enabled?

Focusing on the aerodynamics, what settings have you enabled in AeroDyn and how do these compare to equivalent settings in HAWC2?  

Regardless, I would expect that the aerodynamics routines of HAWC2 have some differences relative to AeroDyn, so, some differences are likely expected.

Best regards,
